docs/docs/creating-and-modifying-pages.md

```diff
@@ -17,7 +17,7 @@ modify pages created in core or plugins or to create client-only pages.
 
 To see what pages are being created by your code or plugins, you can query
 for page information while developing in Graph*i*QL. Paste the following
-query in the Graph*i*QL IDE for your site.
+query in the Graph*i*QL IDE for your site. The graphql API is accessible at `/__graphql`.
```

Comment:
Good call!

I'd assume most people would want to use the IDE not directly query the endpoint.

Perhaps change the language to: "The Graph*i*QL IDE is available when running your sites development server at `HOST:PORT/__graphql` e.g. `localhost:8000/__graphql`."